SDK/AppCenter/Microsoft.AppCenter.UWP/Utils/DeviceInformationHelper.cs

```diff
@@ -55,7 +55,15 @@ protected override string GetSdkName()
         protected override string GetDeviceModel()
         {
             var deviceInfo = new EasClientDeviceInformation();
-            return string.IsNullOrEmpty(deviceInfo.SystemProductName) ? deviceInfo.SystemSku : deviceInfo.SystemProductName;
+            var systemSku = string.IsNullOrEmpty(deviceInfo.SystemSku)
+                || DefaultSystemSku == deviceInfo.SystemSku
+                ? null
```

Comment:
What happens on portal data when both fields are null? Do we still see events and crashes? You can test e2e by hardcoding null and verify. We should make sure that the backend handles that correctly before merging.

Reply:
When we send null, this is displayed as “None” (on Analytics page) and as “Unknown” (on Diagnostics page)